DashAI/front/.eslintrc.js

```diff
@@ -1,23 +1,22 @@
 module.exports = {
-  env: {
-    browser: true,
-    es2021: true,
-  },
-  extends: [
-    'plugin:react/recommended',
-    'plugin:jest/recommended',
-    'airbnb',
-  ],
-  overrides: [
-  ],
-  parserOptions: {
-    ecmaVersion: 'latest',
-    sourceType: 'module',
-  },
-  plugins: [
-    'react',
-  ],
-  rules: {
-    'linebreak-style': ['error', process.platform === 'win32' ? 'windows' : 'unix']
-  },
-};
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "plugin:react/recommended",
+        "standard-with-typescript",
+        "prettier"
+    ],
+    "overrides": [
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+    }
```

Comment:
We need to add this rule for our windows devs. We can't introduce this problem again
```suggestion
    rules: {
    'linebreak-style': ['error', process.platform === 'win32' ? 'windows' : 'unix']
  },
```